docs/en/setup/backend/backend-alarm.md

```diff
@@ -18,34 +18,19 @@ Defines the relation between scope and entity name.
 - **Endpoint Relation**: {Source endpoint name} in {Source Service name} to {Dest endpoint name} in {Dest service name}
 
 ## Rules
-**There are two types of rules: individual rules and composite rules. A composite rule is a combination of individual rules.**
-### Individual rules
 An alerting rule is made up of the following elements:
 - **Rule name**. A unique name shown in the alarm message. It must end with `_rule`.
-- **Metrics name**. This is also the metrics name in the OAL script. Only long, double, int types are supported. See the
-[list of all potential metrics name](#list-of-all-potential-metrics-name). Events can also be configured as the source
-of Alarm. Please refer to [the event doc](../../concepts-and-designs/event.md) for more details.
+- **Expression**. A [MQE](../../api/metrics-query-expression.md) expression that defines the condition of the rule.
+The result type must be `SINGLE_VALUE` and the root operation of the expression must be a [Compare Operation](../../api/metrics-query-expression.md#compare-operation) which provides `1`(true) or `0`(false) result.
+When the result is `1`(true), the alarm will be triggered.
+For example, `avg(service_resp_time / 1000) > 1` is a valid expression for alarm, but `avg(service_resp_time > 1000) + 1` and `service_resp_time > 1000` is not.
```

Comment:
```suggestion
For example, `avg(service_resp_time / 1000) > 1` is a valid expression to indicate the request latency is slower than 1s. The typical illegal expressions are
    - `avg(service_resp_time > 1000) + 1` expression doesn't use `compare operation`
    - `service_resp_time > 1000` expression return a `TIME_SERIES_VALUES` type of values rather than a `SINGLE_VALUE` value.
```